build-logic/src/main/kotlin/dokkabuild.base.gradle.kts

```diff
@@ -21,9 +21,3 @@ tasks.withType<AbstractArchiveTask>().configureEach {
     isPreserveFileTimestamps = false
     isReproducibleFileOrder = true
 }
-
-val integrationTestPreparation by tasks.registering {
-    description =
-        "lifecycle task for preparing the project for integration tests (for example, publishing to the test Maven repo)"
-    group = VERIFICATION_GROUP
-}
```

Comment:
It turns out the `integrationTestPreparation` task wasn't necessary, and `devPublication()` dependencies work across included builds.